docs/src/main/sphinx/security/opa-access-control.md

```diff
@@ -212,6 +218,74 @@ The `targetResource` is used in cases where a new resource, distinct from the on
   }
 }
 ```
+## RowFilters 
+
+Raw filters allow you to denied a group of users to see specifics rows in a table, based on an OPA policy.
+
+for these kind of policies to work, make sure to set `opa.policy.row-filters-uri` in the opa-plugin configuration
+
+here is an example bellow
+```
+            package trino
+            import future.keywords.in
+            import future.keywords.if
+            import future.keywords.contains
+
+            default allow := true
+
+            table_resource := input.action.resource.table
+            is_admin {
+              input.context.identity.user == "admin"
+            }
+
+            rowFilters contains {"expression": "user_type <> 'customer'"} if {
+                not is_admin
+                table_resource.catalogName == "sample_catalog"
+                table_resource.schemaName == "sample_schema"
+                table_resource.tableName == "restricted_table"
+            }
+```
+the format expected by the plugin is a result (here "rowFilters") that contains an array of "{"expressions":"clause"} where each expression is basicaly a where clause.
```

Comment:
```suggestion
The response expected by the plugin is an array of objects, each of them in the format `{"expression":"clause"}`.

Each expression essentially behaves like an additional `WHERE` clause.
```

I would mention two more things:

- It is possible for _multiple_ row filters to be returned for a single OPA request, they will all be applied
- Additionally, each object may contain an `identity` field

The `identity` field allows Trino to evaluate these row filters under a different identity - such that a filter can target a column the requesting user cannot see

Reply:
realy interesting these identity field :) 
thanks !

Reply:
proposal applied